edenai_apis/apis/deepgram/deepgram_api.py

```
@@ -273,6 +273,8 @@ async def audio__atts(
         voice: Optional[str] = None,
         audio_format: str = "mp3",
         speed: Optional[float] = None,
+        speaking_pitch: Optional[int] = None,
```

Ruff (0.14.13)

Unused method argument: `speaking_pitch`

(ARG002)